helm/Chart.yaml

```diff
@@ -24,8 +24,8 @@ version: 0.1.0
 dependencies:
   - name: config-bootstrapper
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.2.37
+    version: 0.2.38
   - name: kafka-topic-creator
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.2.0
+    version: 0.2.x
```

Comment:
0.2.x?

Reply:
Mentioned elsewhere, more idiomatic to write
```suggestion
    version: ~0.2
```

Either way, I'm ok unpinning topic creator and not bootstrapper as you're doing.